File: examples-sdk/go/101c-vilapp-multi-pipeline-benchmark/main.go
// 101c-vilapp-multi-pipeline-benchmark — Go SDK equivalent
// Compile: vil compile --from go --input 101c-vilapp-multi-pipeline-benchmark/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("multi-pipeline-bench", 3090)

	pipeline := vil.NewService("pipeline")
	s.Service(pipeline)

	s.Compile()
}

File: examples-sdk/zig/101c-vilapp-multi-pipeline-benchmark/main.zig
// 101c-vilapp-multi-pipeline-benchmark — Zig SDK equivalent
// Compile: vil compile --from zig --input 101c-vilapp-multi-pipeline-benchmark/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("multi-pipeline-bench", 3090);
    var pipeline = vil.Service.init("pipeline");
    server.service(&pipeline);
    server.compile();
}
